pest_derive/Cargo.toml

```diff
@@ -17,7 +17,8 @@ proc-macro = true
 [dependencies]
 quote = "^0.3"
 syn = "^0.10"
-pest = { git = "https://github.com/pest-parser/pest", branch = "procedural" }
+# pest = { git = "https://github.com/pest-parser/pest", branch = "procedural" }
```

Comment:
It wouldn't build on Travis as well too though, is that ok? I guess it's the issue of depending on a git branch